great_expectations/expectations/core/expect_column_median_to_be_between.py

```diff
@@ -144,8 +145,9 @@ class ExpectColumnMedianToBeBetween(ColumnExpectation):
                     "estimator": "bootstrap",
                     "n_resamples": 9999,
                     "random_seed": None,
-                    "quantile_statistic_interpolation_method": "auto",
-                    "quantile_bias_std_error_ratio_threshold": 0.25,
+                    "quantile_statistic_interpolation_method": "nearest",
+                    "quantile_bias_correction": False,
```

Comment:
@NathanFarmer If we do not yet have a docstring stating that if `quantile_bias_correction` is False, then `quantile_bias_std_error_ratio_threshold` has no effect, then please let us add it associated with those fields where arguments for `NumericMetricRangeMultiBatchParameterBuilder` class are explained.

Reply:
`NumericMetricRangeMultiBatchParameterBuilder` docstring was updated to read:
```
quantile_bias_correction: Applicable only for the "bootstrap" sampling method -- if omitted (default), then
                False is used (bias correction is disabled) and quantile_bias_std_error_ratio_threshold will have no
                effect.
```